### PR TITLE
Adds jobs command

### PR DESCRIPTION
**Pull request by AartiKriplani**
Implements https://github.com/concourse/concourse/issues/605

For now, outputs just the name and paused status, would like to hear opinions on what other attributes of jobs would be helpful. Builds for a job are a separate command anyways.

[#129068087](https://www.pivotaltracker.com/n/projects/1059262/stories/129068087)

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@AartiKriplani Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/fly&pullRequestId=153)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=concourse/fly&pullRequestId=153) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by AartiKriplani** [comment]
Dependent on https://github.com/concourse/go-concourse/pull/5.
Tests will fail until its merged.

**Comment by pivotal-issuemaster** [comment]
@AartiKriplani Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/fly&pullRequestId=153)!

File: commands/jobs.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	err = target.Validate()
+	if err != nil {
+		return err
+	}
+
+	var headers []string
+	var jobs []atc.Job
+
+	jobs, err = target.Team().ListJobs(pipelineName)
+	headers = []string{"name", "pausd"}
```
Comment (jvshahid) [inline commands/jobs.go]: I think this is a typo, `paused` instead of `pausd`

**Comment by chendrix** [comment]
Hi there, I just checked out and merged your PR to `go-concourse`.

Can you recreate this PR without your `.idea` folder in it and we can see if it passes then?

**Comment by pburkholder** [comment]
I would dig being able to use this. Any chance you can revive this PR, @AartiKriplani? Thanks! P.

**Comment by jtarchie** [comment]
Thanks for the PR! We made some changes, but gave you credit for your commits.

**Pull request closed without merging**